app/components/jukebox-onboarding-message/component.js

```diff
@@ -4,20 +4,22 @@ import { inject as service } from '@ember/service';
 const CANCELLED_COOKIE = 'onboardingCancelled'
 
 export default Component.extend({
-  cookies        : service(),
-  showOnboardingMessage: false, 
+  cookies: service(),
+  fastboot: service(),
+  showOnboardingMessage: false,
 
   init() {
     this._super(...arguments);
-    if (!this.get('cookies').exists(CANCELLED_COOKIE)) {
-       this.set('showOnboardingMessage', true);
+    if (this.get('fastboot.isFastBoot') || this.get('cookies').exists(CANCELLED_COOKIE)) {
+      return false;
```

Comment:
one more quick question/point here: why are you using a `return false` here? Is it just to avoid getting to `this.set('showOnboardingMessage', true);`? if this is the case, a cleaner way to do this would be something like: 

```
if (!(this.get('fastboot.isFastBoot') || this.get('cookies').exists(CANCELLED_COOKIE))) {
  this.set('showOnboardingMessage', true);
}
```

**or**

you could just say `return` instead or `return false`, since `init` doesn't typically return a value

Reply:
It was so that the `showOnboardingessage` would definitely not show if there were cookies set and more my/other future devs clarity. The issue in QA that came up before was that it would flicker/render briefly if the user had cookies that cancelled the onboarding message.